scrapy/utils/serialize.py

```diff
@@ -40,7 +40,7 @@ def default(self, o: Any) -> Any:
 class ScrapyJSONDecoder(json.JSONDecoder):
     def __init__(self, *args, **kwargs):
         warnings.warn(
-            "'ScrapyJSONDecoder' is a deprecated setting.\nIt will be removed in a future version of Scrapy.",
+            "'ScrapyJSONDecoder' class is deprecated and will be removed in a future version of Scrapy.",
```

Comment:
```suggestion
            "The 'ScrapyJSONDecoder' class is deprecated and will be removed in a future version of Scrapy.",
```

You could also omit the single quotes, I think, but no strong opinion on that.